private/pkg/command/runner.go

```diff
@@ -96,3 +114,31 @@ func envSlice(env map[string]string) []string {
 	sort.Strings(environ)
 	return environ
 }
+
+type terminator struct {
```

Comment:
Why is this called `terminator` when the interface is called `process`?

Should be moved to a separate file `process.go` and renamed to `process` per style guide.